django-stubs/db/models/base.pyi

```diff
@@ -16,19 +15,17 @@ class ModelState:
     adding: bool = ...
     fields_cache: ModelStateFieldsCacheDescriptor = ...
 
-class ModelBase(type): ...
+class ModelBase(type):
+    @property
+    def _default_manager(cls: Type[_Self]) -> BaseManager[_Self]: ...
```

Comment:
I think that mypy might not like `cls` to be bound to `Type[Model]` 🤔

Reply:
Yes, mypy's right, updated.

Reply:
I have no idea how to solve this, maybe a `type: ignore` should be introduced.
Indeed, there's no guarantee that `ModelBase` would receive `cls` that bounded to `Model`.

Reply:
I think we can fallback to the first solution + `type: ignore`